lib/elixir/lib/file.ex

```diff
@@ -166,6 +166,10 @@ defmodule File do
   It can be regular file, directory, socket, symbolic link, named pipe or device file.
   Returns `false` for symbolic links pointing to non-existing targets.
 
+  ## Options
+
+  If the option `:raw` is set, the file server is not called and only information about local files is returned.
```

Comment:
By reading this, I would try to set `File.exists?(file, raw: true)` which actually doesn't work. So I think we should say something like:

> The list of options currently support a single option, the atom `:raw`, which
> bypasses the file server and only checks for the file locally.